crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MaatCourtDataAssessmentBuilder.java

```diff
@@ -96,6 +91,7 @@ private MaatApiUpdateAssessment buildUpdate(MeansAssessmentDTO assessment) {
                 .withFullTotalAnnualDisposableIncome(assessment.getTotalAnnualDisposableIncome())
                 .withFullOtherHousingNote(meansAssessment.getOtherHousingNote())
                 .withFullTotalAggregatedExpenses(assessment.getTotalAggregatedExpense())
-                .withUserModified(meansAssessment.getUserId());
+                .withUserModified(meansAssessment.getUserId())
+                .withChildWeightings(meansAssessment.getChildWeightings());
```

Comment:
Right now you're just setting the child weightings for update requests. Child weightings should only be updated on INIT assessments, so I think we need an additional check like you've done in the MeansAssessmentService in the classes main function to enable this. Create requests are always INIT assessments, but updates can also be made to them.